### PR TITLE
adding some form functionality

### PR DESCRIPTION
**Pull request by onocy**

### CODE REVIEW

**Review by mboneil10 — APPROVED** [review]

**Comment by mboneil10** [comment]
Why is the deadline overhanging the card? I can't figure out how to change it.

**Comment by mboneil10** [comment]
screen shot 2017-12-03 at 8 56 36 pm

**Comment by mboneil10** [comment]
And should the edit link work? It doesn't work for me unless I manually go to the page. The request isn't going through on the server either... I'll look into this. Meanwhile, I've made the check button work (deletes the chore).

**Comment by mboneil10** [comment]
Okay, update. I've fixed the form so a user can add, update, and delete chores.

File: cleanslate/cleanslate/urls.py [hunk 1]
```diff
@@ -19,12 +19,13 @@
...
 from django.conf import settings
 from django.conf.urls.static import static
+from django.contrib.auth import views as auth_views
 from doma.views import home, profile, calendar, reminders, finance, edit_chore_deadline, create_chore, delete_chore, EditUserProfileView
 
 urlpatterns = [
-    url(r'^$', home, name='doma/login/'),
     url(r'^admin/', admin.site.urls),
     url(r'^doma/$', home, name='doma'),
+    url(r'^$', auth_views.login, name='doma-login'),
```
Comment (L4r1k) [inline cleanslate/cleanslate/urls.py:28]: These lines seem to be in two different pull requests, definitely avoid doing this. Keep new code to just one PR and isolate your projects

File: cleanslate/doma/static/css/base.css [hunk 2]
```diff
@@ -1,15 +1,15 @@
 body {
-     display: flex;
-     min-height: 100vh;
-     flex-direction: column;
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
```
Comment (Cobchise) [inline cleanslate/doma/static/css/base.css]: Again, here is a duplicate

File: cleanslate/doma/static/css/finance_list.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
 
 .card {
-  background: #f1f8e9      ;
+  background: #f1f8e9;
```
Comment (Cobchise) [inline cleanslate/doma/static/css/finance_list.css]: Dupe

File: cleanslate/doma/static/css/logged_out.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.card {
```
Comment (Cobchise) [inline cleanslate/doma/static/css/logged_out.css]: Mostly duplicated

**Review by L4r1k — CHANGES_REQUESTED** [review]
It seems as though this and your other PR are two halves of a whole. Definitely try to isolate your code to avoid this kind of thing.

File: cleanslate/doma/templates/chore_create_form.html [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
 
-    <h1>Create a Chore</h1>
-    
+{% block content %}
+<div class = "card z-depth-5 center">
+    <p>Create a Chore</p>
     <form action="" method="post">
         {% csrf_token %}
         <table>
         {{ form }}
```
Comment (L4r1k) [inline cleanslate/doma/templates/chore_create_form.html:13]: Could add date fields to the form for ease of input

File: cleanslate/doma/templates/chore_delete_form.html [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
+  <link href="{% static 'css/form.css' %}" rel="stylesheet">
+{% endblock %}
 
-{% block content %}
 
+{% block content %}
+<div class = "card z-depth-5 center">
 <h1>Delete Chore</h1>
 
 <p>Are you sure you want to delete this chore?</p>
```
Comment (L4r1k) [inline cleanslate/doma/templates/chore_delete_form.html:12]: The buttons on this and the edit page have no styling, perhaps standardize them like the create page?

**Review by L4r1k — DISMISSED** [review]
Some styling here too

**Comment by L4r1k** [comment]
Other than my comments, chores form works as intended, good job

File: cleanslate/cleanslate/urls.py (hunk 1 above)
Comment (onocy) [reply cleanslate/cleanslate/urls.py:28]: Could you elaborate a bit? 
You're saying this pr includes the code from the previous, correct? If so, what is the best method to "isolate". 
Learning git methodologies like this would be very useful for me, as problems like this seem to continuously arise from my lack of knowledge in this area.

Comment (L4r1k) [reply cleanslate/cleanslate/urls.py:28]: This is why I had offered to teach git earlier in the semester multiple times but unfortunately we never met to do so. I believe you understood this when I explained in slack, regardless, this line shouldn't be necessary in either PR and I have removed it in my most recent PR #73

**Comment by L4r1k** [comment]
Which PR?
